tests/fault_tolerance/deploy/scenario.py

```
@@ -0,0 +1,193 @@
+            logger.info("=" * 60)
+            logger.info("STOPPING EVENTS")
+            logger.info("=" * 60)
+            for event in reversed(events):
+                await event.stop(ctx)
+
+        finally:
+            # Emergency cleanup
+            for event in events:
+                try:
+                    await event.stop(ctx)
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)